java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -89,10 +89,10 @@ public class ColumnVectorTest extends CudfTestBase {
 
   @Test
   void testTransformVector() {
-    try (ColumnVector cv = ColumnVector.fromBoxedInts(2,3,null,4);
+    try (ColumnVector cv = ColumnVector.fromBoxedInts(2,3,4);
```

Comment:
This is a breaking change which has potential impact that may or may not be discovered immediately. I'm a bit nervous with such breaking change merging during the burndown time.

Reply:
Yes, it's a breaking change. We already throw an exception to let the user know nulls aren't supported in the meantime. I was also informed that transforms aren't used by customers presently. 
We still need to rework how nulls are handled in transforms, especially given that we now support multi-column input.

> Do we need this for 25.06?

Yes